### ISSUE TITLE
Power from Beyond (again!) 

### ISSUE THREAD

**Issue by Pickysaurus** [Starfield-Community-Patch/Starfield-Community-Patch#672]
Quest objectives aren't pointing to the correct planet (or anything)

## Details

> This did not resolve my "Power From Beyond" Issue.  I have 3 versions of the quest, and I still have no marker anywhere for them.  No matter how long I explore a planet, I can never find the temples.
> 
> Edit: Actually, I take it back.  I actually have 4 versions of the quest, and this DID solve the problem for 3 of them.  The 1 that it still doesn't show up for is on Altair 4-C (not sure if that matters).
> 
> Edit 2: Now it happened again, this time on Piazzi 4-C.  So, my two "problem quests" are on Altair 4-C and Piazzi 4-C.  Something about "4-C" causing this leftover issue?

Reported by [[kalkano27](https://www.nexusmods.com/starfield/users/2898482)

## Game Version

1.8.86

## Language

English (EN)

## Have you recreated this issue on a new game?

Yes, I can recreate this issue.

## Reproduction Steps

I have one right here, saved right before going into the Unity. I used console commands to give myself the last two remaining powers (Supernova and Alien Reanimation) because if you talk to Vlad, he will ask you to go to Altair IV-C and it's bugged.

In fact, I had 9 powers when he asked me to go to Altair IV-C. So, I avoided talking to him completely and just visited all 119 star systems and collected 22/24 powers (so an additional 13 powers). I then talked to him, and he still asked me to go to Altair IV-C. This is when I gave up and just used console commands for the last two remaining powers.

In that save file, just go to the Lodge and ask Vlad for a temple location, and he'll tell you to go to Altair IV-C. Enjoy figuring this one out. 
[Save4_B722A895_53686570617264_062213_20231123143102_107_0_4.zip](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/files/13453053/Save4_B722A895_53686570617264_062213_20231123143102_107_0_4.zip)

A somewhat earlier save with the same bugs, if this helps. I hadn't given myself the last two powers yet via console commands in this one. (edited)
[Quicksave0_B722A895_53686570617264_062142_20231123010549_107_0_4.zip](https://github.com/Starfield-Community-Patch/Starfield-Community-Patch/files/13453055/Quicksave0_B722A895_53686570617264_062142_20231123010549_107_0_4.zip)

**Comment by Constellation-VASCO** [comment]
On checking the savegame provided it appears that the PFB Quest [000854E8] aliases failed to fill for AnomalyLocation and AnomalyMapMarker

**Comment by Constellation-VASCO** [comment]
Those Aliases do appear to be completely optional - which they probably shouldn't be!

**Comment by Constellation-VASCO** [comment]
Further testing Altair IV-C is not a planet that has a temple on it.

**Comment by Constellation-VASCO** [comment]
I have the same issue too - 2 "power from beyond" quests that do not point to any markers.
Game version 1.8.86 and latest community patch.

**Comment by Constellation-VASCO** [comment]
Try this <https://www.nexusmods.com/starfield/mods/7240/>

**Comment by eugene-kirzhanov** [comment]
> Try this https://www.nexusmods.com/starfield/mods/7240/

I have two bugged "Power from Beyond" quests that don't point to any location on planets: Guniibuu II and Oborum I.
They both have Anomalies but there is no Temple markers on them.
I've been fully surveyed these planets for quite some time so that may be the cause of the problem.

I've used your patch, but got "Fix failed for quest at index 0" (and index 1) messages and then "Power from beyond fix has finished running".

I've added required options to `\Documents\My Games\Starfield\StarfieldCustom.ini` but no logs were created in `\Documents\My Games\Starfield\Logs\Script\User`. just like the folder was not created too (also I've tried to create a folder but it didn't help).

I can't upload my save game here due to file size limit, so I've uploaded it to Google Drive: https://drive.google.com/file/d/1H1aN5RSLhz_Z4njy7_gcuulNYkN-B8tg
